### PR TITLE
test(ProgressCircular): Improve test coverage

### PR DESCRIPTION
**Pull request by laraharrow**
Thank so much for contributing to @looker/components!

The CI & [README](https://github.com/looker-open-source/components/blob/main/README.md) should help to make most of our code standards clear

Before requesting review from another developer please delete the lines above and summarize your change.

### Requirements

Please check the following items are addressed in your pull request (or are not applicable)

- [x] Includes test coverage for all changes
- [ ] Documentation updated
- [ ] i18n impacts
- [ ] a11y impacts

### CODE REVIEW

**Review by mdodgelooker — APPROVED** [review]
LGTM

**Review by mdodgelooker — CHANGES_REQUESTED** [review]
Ok I'm stumped on where the 88.89% on `DeterminateProgress.tsx` is coming from. The only tiny piece that's not being tested is the `progress = 0` default value. In reality, `progress` is never undefined, so I think you could remove the ` = 0` as the `?` in `progress?: number` in the props.

File: packages/components/src/ProgressCircular/ProgressCircular.test.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+  test('renders progress of 25', () => {
+    renderWithTheme(<ProgressCircular progress={25} />)
+    expect(screen.queryByRole('progressbar')).toHaveAttribute(
+      'aria-valuenow',
+      '25'
+    )
+  })
+
+  test('renders progress of 0', () => {
```
Comment (laraharrow) [inline packages/components/src/ProgressCircular/ProgressCircular.test.tsx:46]: @mdodgelooker here is the test I wrote for testing when progress = 0

**Review by mdodgelooker — APPROVED** [review]
@laraharrow because the progress prop is never undefined by the time it renders DeterminateProgess, the default in that component is never used. That's the only place I can see that isn't covered but I don't think it really matters anyway.

**Pull request merged**